src/Rezoom.SQL.Compiler/InferredTypes.fs

```diff
@@ -280,6 +284,10 @@ and [<NoComparison>]
     /// Resolve a reference to a table which may occur as part of a TableExpr.
     /// This will resolve against the database model and CTEs, but not table aliases defined in the FROM clause.
     member this.ResolveObjectReference(name : ObjectName, inferView) =
+        match this.Model.Schema (Some name.ObjectName) with
```

Comment:
@rspeele: This is simplest thing that seems to work.

I'm prepending any object resolution here, which may have an impact (no test failing though), should I only do that when the `name.SchemaName` is empty? or should it be handled another way?